examples/async_example.py

```diff
@@ -0,0 +1,81 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import time
+from rev_ai import apiclient, streamingclient
+from rev_ai.models import MediaConfig
+
+#String containing your access token
```

Comment:
For all of these comments, please add a space between the `#` and the rest of the comment. It makes it much more readable